src/services/Common/configuration/Urls.ts

```diff
@@ -0,0 +1,18 @@
+export class RouterUrls {
+    public static login = "/login";
+    public static suggest = "/suggest";
+    public static user = "/user";
+    public static book = "/book";
+    public static detect = "/detect";
+    public static stats = "/stats";
+    public static rewards = "/rewards";
+}
+
+export class UserUrls {
+    public static base = "/users";
+    public static me = "/users/me";
+}
+
+export class AuthenticateUrls {
+    public static base = "/authenticate";
```

Comment:
C'est plus le bon endpoint ça 
```suggestion
    public static base = "/auth";
```

Reply:
Merci ahah